airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/BaseDatabaseConfigPersistence2Test.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.config.persistence;
+
+import static org.jooq.impl.DSL.asterisk;
+import static org.jooq.impl.DSL.count;
+import static org.jooq.impl.DSL.table;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.airbyte.config.ConfigSchema;
+import io.airbyte.config.StandardDestinationDefinition;
+import io.airbyte.config.StandardSourceDefinition;
+import io.airbyte.config.StandardSourceDefinition.SourceType;
+import io.airbyte.db.Database;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.UUID;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.jooq.Record1;
+import org.jooq.Result;
+import org.jooq.Table;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+/**
+ * This class provides downstream tests with constants and helpers.
+ */
+public abstract class BaseDatabaseConfigPersistence2Test {
```

Comment:
The test classes `BaseDatabaseConfigPersistence2Test`, `DatabaseConfigPersistence2LoadDataTest` , `DatabaseConfigPersistence2MigrateFileConfigsTest`, `DatabaseConfigPersistence2Test`  are inspired by the existing tests `BaseDatabaseConfigPersistenceTest`, `DatabaseConfigPersistenceLoadDataTest`, `DatabaseConfigPersistenceMigrateFileConfigsTest` , `DatabaseConfigPersistenceTest`